### ISSUE TITLE
Aria issue on aria-dialog for time picker

### ISSUE THREAD

**Issue by endorphines** [Vuepic/vue-datepicker#941]
**Describe the bug**
Our accessibility scanner reports that when the time picker dialogs are open, there is no aria-label, and throws a violation for `ARIA dialog and alertdialog nodes should have an accessible name`. I tried to see if there is an existing matching aria-label but doesn't seem to be available.

**To Reproduce**
Run any accessibility scanner when the time picker dialog is opened.

**Expected behavior**
To have aria-label on time picker dialogs or allow us to set aria-labels for time picker dialogs

**Screenshots**
![Screenshot 2024-07-15 at 1 53 58 PM](https://github.com/user-attachments/assets/2c62f80f-a4df-474c-a3e6-1f9da0267f63)

**Desktop & mobile (please complete the following information):**
 - Browser: Chrome v126
 - Library version: 8.8.1

Thank you!

**Comment by Jasenkoo** [comment]
Already added, will be in the next release, there are quite a few accessibility improvements

**Comment by endorphines** [comment]
That's awesome! Will be waiting for the next release. Thank you very much!
